Button component

markonastic opened the pull request.

markonastic committed:
Created reusable button component,
button component implemented in other components

markonastic committed:
hover event in footer arrow button set to false

luckySeventeen commented on src/app/common/button/button.component.ts:
```diff
@@ -0,0 +1,36 @@
+})
+
+export class ButtonComponent {
+
```
One boolean propertije koji dodaju css klase sto sam ti napisao gore u komentaru najjednostavnije sve ovde definises sa input dekoratorima.

luckySeventeen replied:
ovde definises input `@Input() isFull: boolean = false`;

luckySeventeen commented on src/app/common/button/button.component.scss:
```diff
@@ -0,0 +1,14 @@
+  cursor: pointer;
+  transition: background-color ease-in-out 0.3s,
+              border-color ease-in-out 0.3s;
+}
```
dodas ovde .is-full klasu sa svojim hover stateom

luckySeventeen commented on src/app/contact/contact.component.html:
```diff
@@ -27,13 +27,20 @@ <h3>Have a question or want to work together?</h3>
     </div>
       <div *ngIf="emailSuccess" class="alert alert-success">Your message was sent successfully.</div>
-    <input type="submit" value="Submit" class="btn">
+    <app-button class="btn"
```
I ovde de pozivas button dodas `[isFull] = true` i zavrsis sve.

markonastic committed:
Edited button component,
removed ngStyle and added ngClass  for styling buttons,
Also edited all components that used app-button and applied to new settings

luckySeventeen commented on src/app/common/button/button.component.html:
```diff
@@ -0,0 +1,21 @@
+<div class="button"
+     [ngClass]="{
+        'white-aqua': btnType === 'whiteAqua',
```
Ovo nema poentu brate, ja sam bio u fazonu da napravis dve tri varijacije gudmeta po principu da li ima border ili da li ima background ili da li je veliko ili malo, a ne da imas varijaciju za svaku boju backgrounda ili boju bordera....
